### PR TITLE
CWG2345 Jumping across initializers in init-statements and conditions

### PR DESCRIPTION
**Pull request by jensmaurer**

### CODE REVIEW

File: source/statements.tex [hunk 1]
```diff
@@ -898,7 +898,9 @@
 \indextext{initialization!jump past}%
 \indextext{\idxcode{goto}!initialization and}%
 It is possible to transfer into a block, but not in a way that bypasses
-declarations with initialization. A program that jumps\footnote{The transfer from the condition of a \tcode{switch} statement to a
+declarations with initialization (including such in \grammarterm{condition}s
```
Comment (jwakely) [inline source/statements.tex]: I find this use of "such" to be a little awkward.

Comment (jensmaurer) [reply source/statements.tex]: Let's try "ones" instead; Google found a similar sentence "Dozens of Vegetable Products, Including Ones Sold at Trader Joe's and Walmart, Recalled Over Listeria Concerns".

Comment (jwakely) [reply source/statements.tex]: Yes, I like that more. Another option that doesn't sound wrong to me is "including such declarations in ..."

**Comment by jensmaurer** [comment]
CWG in Jacksonville: add in 9p1: "A condition that is not an expression is a declaration ([dcl])." Proceed editorially.

**Comment by jensmaurer** [comment]
Updated for CWG comment.

**Pull request merged**